### ISSUE TITLE
Data Import - Missing landmark markup and semantics

### ISSUE THREAD

**Issue by yan-ilumino** [mine-cetinkaya-rundel/accessible-cheatsheets#2]
**Page**: https://mine-cetinkaya-rundel.github.io/accessible-cheatsheets/data-import/data-import-accessible.html

**Issue**:

The document has an obvious header and footer, but they are not markup as such. The lack of header and footer landmark might prevent screen reader users from understanding and navigating the webpage and its information. 

**How to fix it**

Mark up different regions of web pages and applications, so that they can be identified by web browsers and assistive technologies.

**Success Criteria**

- [1.3.1 Info and Relationships (Level A)](https://www.w3.org/WAI/WCAG21/quickref/#info-and-relationships)
